### PR TITLE
Deprecate Faker.Name in favor of Faker.Person

### LINKED ISSUE #316: Move Faker.Name to Faker.Person
I'm not sure the `Name` namespace makes sense. All the `Faker.Name` functions seem to apply specifically to naming people. I propose deprecating `Faker.Name` and replacing it with `Faker.Person`. I believe this would also be more inline with other modules that deal with naming like `Faker.Company`.

### PR DESCRIPTION
**Pull request by anthonator**
Closes #316 

- [x] USAGE.md
- [x] CHANGELOG.md

### CODE REVIEW

**Comment by anthonator** [comment]
@igas @vbrazo what are your thoughts on this?

**Comment by anthonator** [comment]
Hey team. Any way I can get some feedback on this?

File: USAGE.md [hunk 1]
```diff
@@ -69,15 +69,15 @@
...
-- [Faker.Name.Es](lib/faker/name/es.ex)
-- [Faker.Name.Hy](lib/faker/name/hy.ex)
-- [Faker.Name.It](lib/faker/name/it.ex)
-- [Faker.Name.PtBr](lib/faker/name/pt_br.ex)
 - [Faker.Nato](lib/faker/nato.ex)
   <!-- O -->
   <!-- P -->
+- [Faker.Person](lib/faker/name.ex)
+- [Faker.Person.En](lib/faker/name/en.ex)
+- [Faker.Person.Es](lib/faker/name/es.ex)
```
Comment (vbrazo) [inline USAGE.md:77]: I wonder if we should rename the folder name to `person`.

```suggestion
- [Faker.Person.Es](lib/faker/person/es.ex)
```

Comment (anthonator) [reply USAGE.md]: Good call!

File: lib/faker/name.ex [hunk 2]
```diff
@@ -1,8 +1,8 @@
 defmodule Faker.Name do
-  import Faker, only: [localize: 1]
+  alias Faker.Person
 
   @moduledoc """
-  Functions for generating names
+  Deprecated. Use Faker.Person instead.
```
Comment (vbrazo) [inline lib/faker/name.ex:5]:
```suggestion
  Deprecated. Faker.Name will be removed in 1.0.0. Please use Faker.Person instead.
```

**Pull request merged**